dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/ApiAccessor.kt

```diff
@@ -313,6 +313,20 @@ class ApiAccessor {
         return UploadInfo(testCompanyInformation, companyDataControllerApi.postCompany(testCompanyInformation))
     }
 
+    fun uploadOneCompanyWithIdentifiers(
+        lei: String? = null,
```

Comment:
Are those correct? Should the three different identifiers be handled differently?
Also see my comment to GeneralDataProvider.kt

Reply:
@PaulChr99

Reply:
If I understood it correctly, both lei and permId are unique identifiers of a company, while there may well be multiple isins for one and the same company. I think the company upload in the backend thus handles all identifiers as List<String>, even though this does not make sense for some of them from business perspective. However, compared to here, we have one field for identifiers in the backend, holding a map of type Map<IdentifierType, List<String>>. As it was no major issue to make that distinction here, I aligned to that business logic. However, I may also revert that depending on whether anyone has a strong opnion on that

Reply:
ok